virttest/utils_libvirt/libvirt_secret.py

```diff
@@ -3,12 +3,62 @@
 """
 
 import logging
+import re
+
+from aexpect import remote
+
+from avocado.core import exceptions
 
 from virttest import virsh
+from virttest.utils_test import libvirt
+from virttest.libvirt_xml import secret_xml
 
 LOG = logging.getLogger('avocado.' + __name__)
 
 
+def create_secret(sec_dict=None, remote_args=None):
```

Comment:
Could someone elaborate more details why we pass monolithic sec_dict here? What benefits it can bring really?

Reply:
as the deafult value ,
```
    if not sec_dict:
        sec_dict = {'secret_ephemeral': 'no',
                    'secret_private': 'yes',
                    'description': 'secret desc',
                    'usage': 'volume',
                    'volume': '/path/to/volume'}
```